app/models/information_request.rb

```diff
@@ -87,22 +87,22 @@ def information_required
     info << "Legal Aid Agency" if laa?
     info << "Office of the Public Guardian (OPG)" if opg?
     info << "Somewhere else in the Ministry of Justice" if moj_other?
-    info.join(", ")
+    info.join("<br>").html_safe
```

Comment:
You've added br tags to three places here, but you're only removing them from one place in the email view.